runtime/moonbase/src/lib.rs

```diff
@@ -417,6 +418,9 @@ parameter_types! {
 	///     (max_extrinsic.ref_time() / max_extrinsic.proof_size()) / WEIGHT_PER_GAS
 	/// )
 	pub const GasLimitPovSizeRatio: u64 = 4;
+	/// The amount of gas per storage (in bytes).
+	pub GasLimitStorageGrowthRatio: u64 =
+		BlockGasLimit::get().low_u64().saturating_div(BLOCK_STORAGE_LIMIT);
```

Comment:
This expression is computed on block exution, please use a constant instead, and maybe create a rust unit test to check that the constant match the formula

Reply:
Done